src/script/components/Modals/PrimaryModal/PrimaryModal.tsx

```diff
@@ -55,7 +55,7 @@ export const PrimaryModalComponent: FC = () => {
   const isModalVisible = currentId !== null;
   const passwordValueRef = useRef<HTMLInputElement>(null);
   const [isFormSubmitted, setIsFormSubmitted] = useState(false);
-  const isBackupPasswordValid = useMemo(() => passwordInput && isValidPassword(passwordInput), [passwordInput]);
+  const isBackupPasswordValid = useMemo(() => passwordInput === '' || isValidPassword(passwordInput), [passwordInput]);
```

Comment:
Hmm.. this can be checked like that :) 

```suggestion
  const isBackupPasswordValid = useMemo(() => !passwordInput || isValidPassword(passwordInput), [passwordInput]);
```

Reply:
I think we explicitly need to check if the password is empty. Either is can be empty or follow the regex rule.

Reply:
Sure, we can check it like that also, no big difference to me in this case. Is it some sort of blocker for you or just "nice to have" 😄?

Reply:
Just a wild guess 👐 
@olafsulich, can put some light on it.

Reply:
Oh I meant, is it a blocker for @przemvs 😅

Reply:
Nah, just suggestion 🤣